README.md

```diff
@@ -65,138 +30,36 @@ After you're finished please remove all the comments and instructions!
 
 > The very first project of the 1st module of microverse full stack web development program
```

Comment:
- [ ] Just as the previous reviewer mentioned, please remove any trace of template that is not personalized from your ReadMe. This comment was intended as guide for you on what to add to this section but it has no relevance other users interacting with you project. That is why it is good to always delete lines like this in your ReadMes.

Reply:
All braces removed